### ISSUE TITLE
QE: superconductive qubits

### ISSUE THREAD

**Issue by gamberoillecito** [quantum-engineering-polito/CourseNotes#59]

**Comment by p3drotto** [comment]
1. line 41. O.5K is the temperature at which the thermal energy is equal to the spacing between the energy levels: h omega = KT -> T= h omega/K. To avoid spontaneous transition from |0> to |1> due to thermal excitation, you need to operate the qubit at a much lower temperature typically of the order of 10mK (reached with dilution refrigerators)
2. line 51. Write that the classical analogy holds identifying Q as the momentum and phi as the position. 
3. line 57. value -> eigenvalues. as a physicist I would rephrase with something like: "to find the eigenvalues of H we can procede by  writing Q and Phi as linear combinations of the ladder operators".
4. line 138. I wouldn't call n "the number operator"- The number operator is the one that count the level in an harmonic oscillator: a+a. (+ = dagger). 
5. line 153. This is not the trasmont regime. The trasmont regime is actually the one where Ej>>Ec. 
6. line 172. I would add the plots on hanarmonicity and chart dispersion and comment that increasing the Ej/Ec tario: The charge dispersion decreases exponentially while the anharmonicity decreases as a square root
7. Line 185. I would add also that since Ec << Ej, the "kinetic energy"(associated to the capacitance) is much smaller than the "potential energy" (associated to the inductance)
8. line 214. Is the rotating Wave approximations. The idea Is that the terms with different a different number of b and b+ corresponds to higher frequency terms, that can be neglected. I do not know the details about this approximations, in the papers they justify it in this way.
9.line  218. add that The second term is the one that gives the anharmonicity, that makes so that E01 is different than E12.

**Comment by p3drotto** [comment]
10. At the end we can add few words about the part on coherent control. Just put the graph and add the comment that the prof made

**Comment by BabelDev0** [comment]
ok I checked the note and added the considerations of @p3drotto

**Comment by BabelDev0** [comment]
please @gamberoillecito check it
